src/tox_lsr/config_files/tox-default.ini

```diff
@@ -238,23 +238,21 @@ commands =
     bash {lsr_scriptdir}/runshellcheck.sh
     {[lsr_config]commands_post}
 
-[lsr_ansible-lint]
-configfile = {lsr_configdir}/ansible-lint.yml
-
 [testenv:ansible-lint]
 changedir = {toxinidir}
-deps =
-    {env:LSR_ANSIBLE_LINT_DEP:ansible-core==2.14.*}
-    ansible-lint=={env:LSR_ANSIBLE_LINT_VER:5.*}
+whitelist_externals =
+    podman
+    bash
 commands_pre =
     bash {lsr_scriptdir}/ansible-lint-helper.sh pre
 commands =
     bash {lsr_scriptdir}/setup_module_utils.sh
     {[lsr_config]commands_pre}
-    ansible-lint -v --exclude=tests/roles --exclude=.github \
-        --exclude=.yamllint.yml \
-        -c {[lsr_ansible-lint]configfile} \
-        {env:RUN_ANSIBLE_LINT_EXTRA_ARGS:} {posargs}
+    {env:LSR_CONTAINER_RUNTIME:podman} run --rm --privileged \
```

Comment:
We may want to add the description to README.md. But it's a very minor issue...

> By default, it will use podman to run the container. You can set LSR_CONTAINER_RUNTIME to use a different runtime

Reply:
updated
also deprecated several old env. vars.